ansible/roles/atmo-kanki-irodsclient/vars/Ubuntu.yml

```diff
@@ -12,6 +12,6 @@ dependencies:
   - g++
 
 irods_files:
-  - ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-runtime-4.1.9-ubuntu14-x86_64.deb
-  - ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-icommands-4.1.9-ubuntu14-x86_64.deb
-  - ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-dev-4.1.9-ubuntu14-x86_64.deb
+  - { url: 'ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-runtime-4.1.9-ubuntu14-x86_64.deb', dest: '/tmp/irods-runtime.deb' }
```

Comment:
Is this FTP download not protected with TLS? If it's not, then this represents a security issue, because we aren't checking that a man in the middle isn't serving us a different (possibly malicious) .DEB package.

[get_url](https://docs.ansible.com/ansible/latest/modules/get_url_module.html?highlight=get_url) has a `checksum` parameter. I think we should hard-code the checksums in this dictionary like this, and verify them in the download task:

```
{ url: 'ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-icommands-4.1.9-ubuntu14-x86_64.deb', dest: '/tmp/irods-icommands.deb', sha1sum: 'deadbeefdeadbeefdeadbeefdeadbeefdeadbeef' }
```

(Incidentally, the iRODS consortium [doesn't serve their checksums via TLS](ftp://ftp.renci.org/pub/irods/releases/4.1.11/checksums.txt), which itself is an issue.)

Reply:
Good idea. Did you get that shasum by downloading and checking it yourself or did you find one posted somewhere?

Reply:
Never mind, I found them!

Reply:
Don't let that hold up this PR, I'll submit another to fix when this is merged.